### ISSUE TITLE
Error in build

### ISSUE THREAD

**Issue by CesarDenis** [ChilliCream/react-rasta#42]
I am using CRA and giving the `yarn build` command displays the following error:

```bash
Failed to compile.

Failed to minify the code from this file:

        ./node_modules/react-rasta/lib/theme/StyledComponents.js:11

Read more here: http://bit.ly/2tRViJ9
```

Can you help me?
Best regards

**Comment by rstaib** [comment]
Actually, this is an issue of _Create React App_. Here are the good news: they already fixed it in the [next](https://github.com/facebook/create-react-app/blob/next/packages/react-scripts/config/webpack.config.dev.js#L259-L285) branch. So we probably see this fix soon - hopefully.

In the meantime I am going to provide _ES5_ as additional version. The _main_ version is still _ES6_, but you would be able to explicitly link the _ES5_ version in your code.

Please let me know what you think about it.

**Comment by CesarDenis** [comment]
I think it's a good alternative until the next version of CRA.

I made a fork and changing the target to ES5, to build for production output.

**Comment by rstaib** [comment]
Okay cool. I guess I'm going to create a new version this evening today which contains both _ES5_ and _ES6_ precompiled versions.

**Comment by rstaib** [comment]
Version 1.0.3 is on its way.

Use `import { ... } from 'react-rasta';` for _ES6_ and `import { ... } from 'react-rasta/lib/es5';` for _ES5_.

**Comment by rstaib** [comment]
Had some issues with the deployment and therefore unpublished version 1.0.3 and instead published a new version which is now 1.0.4

**Comment by CesarDenis** [comment]
Tks @rstaib
